src/fe-common/core/fe-windows.c

```diff
@@ -42,15 +42,25 @@ static int daytag;
 static int daycheck; /* 0 = don't check, 1 = time is 00:00, check,
                         2 = time is 00:00, already checked */
 
+static GSequenceIter *windows_seq_begin(void)
```

Comment:
If you could stash all the `static` helpers together at the top of the file it'd be tidier.